python/ray/data/context.py

```diff
@@ -247,6 +249,8 @@ class DataContext:
         log_internal_stack_trace_to_stdout: Whether to include internal Ray Data/Ray
             Core code stack frames when logging to stdout. The full stack trace is
             always written to the Ray Data log file.
+        raise_original_exception: Whether to raise the original exception encountered
+            in user-defined functions instead of wrapping it in a `UserCodeException`.
```

Comment:
could you also add details about this new attribute to [this docs page](https://github.com/ray-project/ray/blob/master/doc/source/data/execution-configurations.rst?plain=1#L62)? a short one-line explanation is good, basically the same content as your description here. thanks!

Reply:
fixed!